### PR TITLE
Update README for clarity and to remove redundancy

### PR DESCRIPTION
**Pull request by keyanp**
Addresses some concerns presented in issue #1157, although I don't think it is necessary to add click to the README.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -19,11 +19,10 @@
 
     ~ What do I need?
 
-      Jinja 2.4 and Werkzeug 0.7 or later.
-      `pip` or `easy_install` will install them for you if you do
-      `pip install Flask`.  I encourage you to use a virtualenv.
-      Check the docs for complete installation and usage
-      instructions.
+      If you do `pip install Flask`, `pip` or `easy_install` will
```
Comment (untitaker) [inline README]: That sentence is a bit warped (has always been), executing `pip` doesn't somehow launch `easy_install`.

Comment (keyanp) [inline README]: How about: 

All dependencies are installed by using `pip install Flask`.

Comment (untitaker) [inline README]: sure, go ahead

**Comment by untitaker** [comment]
Right on, thanks!

**Pull request merged**